### PR TITLE
Restructure front back

### PR DESCRIPTION
**Pull request by DiegoManzanarezx**
The intention of this pull request is to:

- Restructure the repository in order to have a **test-runner-backend** folder and a **test-runner-frontend** folder.

- Set one gitignore file inside each folder and one at the root level.

### CODE REVIEW

**Comment by DanElliottPalmer** [comment]
👍

File: test-runner-backend/src/main/java/graphql/testrunner/service/TestRunnerService.java [hunk 1]
```diff
@@ -12,8 +12,6 @@
 
 import static java.util.Objects.nonNull;
 
-import static graphql.testrunner.document.Status.FINISHED;
-import static graphql.testrunner.document.Status.RUNNING;
```
Comment (DanElliottPalmer) [inline test-runner-backend/src/main/java/graphql/testrunner/service/TestRunnerService.java:16]: should this be removed?

Comment (DiegoManzanarezx) [reply test-runner-backend/src/main/java/graphql/testrunner/service/TestRunnerService.java:16]: Those are unused imports only

**Review by DanElliottPalmer — APPROVED** [review]
FOUND THE APPROVE BUTTON!

**Comment by adarsh-jaiswal** [comment]
The workflow.yml file has a startup-script that needs to be adjusted to accommodate the change in directory.
**current**-    value: "#! /bin/bash \ncd tmp\nmkdir test-runner\ncd test-runner\npwd\nls\ngit clone https://github.com/adarsh-jaiswal/graphql-java-test-runner.git\ncd graphql-java-test-runner/ \ndocker build --no-cache -t test-runner .\ndocker run --publish 80:8080 test-runner"
**update** -                             value: "#! /bin/bash \ncd tmp\nmkdir test-runner\ncd test-runner\npwd\nls\ngit clone https://github.com/adarsh-jaiswal/graphql-java-test-runner.git\ncd graphql-java-test-runner/**\ncd test-runner-backend** \ndocker build --no-cache -t test-runner .\ndocker run --publish 80:8080 test-runner"

**Note**: When we move this to graphql-java repository, we need to change the github url.

**Pull request merged**